modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue

```diff
@@ -240,13 +246,8 @@ export default {
     margin-bottom: 0;
   }
 
-  .apos-field__info,
-  .apos-input-wrapper {
-    width: 48%;
-  }
```

Comment:
Not sure it's a good thing to remove totally this. Or maybe we can add a `flex-shrink: 1`.